test/pathforaSpec.js

```diff
@@ -1583,6 +1583,220 @@ describe('API', function () {
     expect(widget.length).toBe(0);
   });
 
+  it('should show widget using simple match', function () {
+    var form1 = new pathfora.Form({
+      id: '88ee86cf72b44e67bf758cc743ac1a5d',
+      msg: 'subscription',
+      headline: 'Header',
+      layout: 'slideout',
+      position: 'bottom-right',
+      displayConditions: {
+        urlContains: [
+          {
+            match: 'simple',
+            value: 'localhost/context.html'
+          }
+        ]
+      }
+    });
+    var form2 = new pathfora.Form({
+      id: 'a793b7352c3346e493573a6827be7815',
+      msg: 'subscription',
+      headline: 'Header',
+      layout: 'slideout',
+      position: 'bottom-right',
+      displayConditions: {
+        urlContains: [
+          {
+            match: 'simple',
+            value: 'localhost/context'
+          }
+        ]
+      }
+    });
+    pathfora.initializeWidgets([ form1, form2 ]);
+
+    var widget = $('#' + form1.id);
+    expect(widget.length).toBe(1);
+
+    var widget = $('#' + form2.id);
+    expect(widget.length).toBe(0);
+  });
+
+  it('should show widget using exact match', function () {
+    var form1 = new pathfora.Form({
+      id: 'e71c5416ac7345bcba8c5330d14c4a2e',
+      msg: 'subscription',
+      headline: 'Header',
+      layout: 'slideout',
+      position: 'bottom-right',
+      displayConditions: {
+        urlContains: [
+          {
+            match: 'exact',
+            value: 'http://localhost:9876/context.html'
```

Comment:
Idk if this will come up, but maybe make a comment somewhere here that if the test fails it could be due to karma not using the default port? :/